.github/workflows/integration_tests.yml

```
@@ -381,6 +381,7 @@ jobs:
               "$JMETER_SERVICE_HELM_CHART_NAME"
           else
             helm install -n ${KEPTN_NAMESPACE} keptn ${HELM_CHART_NAME} \
+              $(if [[ $HELM_CHART_NAME == *"dev"* ]]; then echo "-f ./test/assets/helm-charts/keptn/values-int-tests.yaml"; fi) \
```

Please add some quotes here, just in case the variable is unset for some reason
```suggestion
              $(if [[ "$HELM_CHART_NAME" == *"dev"* ]]; then echo "-f ./test/assets/helm-charts/keptn/values-int-tests.yaml"; fi) \
```